src/components/shared/avatar/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { styled } from '@mui/material/styles';
+import Badge from '@mui/material/Badge';
+import Avatar, { AvatarProps } from '@mui/material/Avatar';
+
+import LOGO_IMG_SRC from '@/assets/Logo96.svg';
+
+interface BasicAvatarProps extends AvatarProps {
+  imgSrc?: string;
+  alt?: string;
+  width?: number;
+  height?: number;
+  isUserOn?: boolean;
+}
```

Comment:
저희 컴포넌트 Prop 은 컴포넌트 파일 내부에서만 local 하게 사용되므로,
export 하지 않아도 되는 경우에는 Props 라고 네이밍하기로 했던 것 같은데, 확인 부탁드려도 될까요?
```suggestion
interface Props extends AvatarProps {
  imgSrc?: string;
  alt?: string;
  width?: number;
  height?: number;
  isUserOn?: boolean;
}
```

Reply:
앗 회의록 확인해보니 Props가 맞네요 반영하겠습니다!